src/Core/RemoteFSProtocol.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+namespace DB
+{
+
+namespace RemoteFSProtocol
+{
+    enum PacketType
```

Comment:
why not make a differentiation between client and server packets (similar as in `Core/Protocol.h`)? it will be better because here looks like 90% of values are client packets.